sorc/ncep_post.fd/MDL2P.f

```diff
@@ -4082,13 +4083,18 @@ SUBROUTINE MDL2P(iostatusD3D)
          if(me==0)PRINT*,'CALLING SLR'
          egrid1=spval
          call calslr_roebber(TPRS,RHPRS,EGRID1)
+      !   call calslr_uutah(EGRID1)
 !$omp parallel do private(i,j) 
          do j=jsta,jend
          do i=ista,iend
+            grid1(i,j)=spval
             if(egrid1(i,j) < spval) then
-                grid1(i,j)=1000./egrid1(i,j)
-            else
-                grid1(i,j)=spval
+              if(egrid1(i,j)>=1.) then
+                 grid1(i,j)=1000./egrid1(i,j)
+              else
+                 grid1(i,j)=spval
+              endif
+            !  grid1(i,j)=egrid1(i,j)
```

Comment:
@JesseMeng-NOAA Is this line for UTA SLR output?

Reply:
@JesseMeng-NOAA I would like to run a testing for UTA SLR. The UTA SLR should be assigned grid1=egrid1 or grid1=1000./egrid1?

Reply:
@JesseMeng-NOAA Got it. Thanks!